### PR TITLE
modify random_constellation

### PR DESCRIPTION
**Pull request by stvreumi**

### CODE REVIEW

**Comment by stvreumi** [comment]
If I move too much code to `dataAccessObject.py`, please let me know. I will change it.

**Comment by Billy4195** [comment]
I think the more intuitive thought of getFortune() is pass `date` and `constellation`, the other parts of this function has less association with ***getting fortune***.

File: dataAccessObjects.py [hunk 1]
```diff
@@ -237,3 +237,23 @@ def getArrangeMode(self):
             #TODO check need to raise exception or not
             return None
 
+class FortuneDao(DefaultDao):
+    def getFortune(self):
+        with UserDao() as userDao:
+            date = userDao.getUserBirthday()
```
Comment (Billy4195) [inline dataAccessObjects.py]: getUserBirthday() can be extracted from this function.

File: dataAccessObjects.py [hunk 2]
```diff
@@ -237,3 +237,23 @@ def getArrangeMode(self):
...
+        if date == None:
+            constellation = ['摩羯座','水瓶座','雙鱼座','白羊座','金牛座','雙子座','巨蟹座','獅子座','處女座','天秤座','天蝎座','射手座']
+            constellation = random.choice(constellation)
+        else:
+            constellation = Zodiac(date.month,date.day)
+
+        today=str(datetime.date.today())
```
Comment (Billy4195) [inline dataAccessObjects.py]: The above code to get constellation can also be extracted.

**Comment by stvreumi** [comment]
Ok, I finish modification. Please help me review, thanks.

**Pull request merged**